docs/vectorizer-worker.md

```diff
@@ -0,0 +1,264 @@
+   ```
+   name: pgai
+   services:
+     vectorizer-worker:
+       image: timescale/pgai-vectorizer-worker:0.1.0rc4
+       environment:
+         PGAI_VECTORIZER_WORKER_DB_URL: postgres://<username>:<password>@<host>:<port>/<database-name>
+         OPENAI_API_KEY: <your-api-key>
+   ```
+
+  1. Replace the values of:
+     - `PGAI_VECTORIZER_WORKER_DB_URL`: the postgres connection string to the database where you have defined vectorizers.
+     - `OPENAI_API_KEY`: with a key from your AI provider.
+
+1. **Run the vectorizer worker**
+
+   For self-hosted, you run a pgai vectorizer worker to automatically create embedding from the data in your
+   database using [vectorizers you defined previously](/docs/vectorizer.md#define-a-vectorizer).
+
+   In a new terminal, start the vectorizer worker:
+   ```shell
+   docker-compose up -d vectorizer-worker
+   ```
+
+   You can also use the run command
+   `docker run timescale/pgai-vectorizer-worker:0.1.0rc4 --db-url <Same value as PGAI_VECTORIZER_WORKER_DB_URL>`
+
+### Install vectorizer worker in your local environment
+
+On your local machine:
+
+1. **Install [pgai](https://pypi.org/project/pgai/) from PyPI**
+
+   ```bash
+   pip install pgai
+   ```
+
+   The vectorizer worker, `pgai vectorizer worker` is now in your `$PATH`.
+
+1. **Run the vectorizer worker**
+
+    After you [define a vectorizer in your database](/docs/vectorizer.md#define-a-vectorizer), you run 
+    a vectorizer worker to generate and update your embeddings:
+
+    1. Set the connection string to your self-hosted database and the API key for the external embeddings
+      provider as environment variables:
+
+       ```bash
+       export PGAI_VECTORIZER_WORKER_DB_URL="postgres://<user>:<password>@<host>:<port>/<dbname>"
+       export OPENAI_API_KEY="Your OpenAI API key"
+       ```
+
+    1. Run the vectorizer worker:
+
+       ```bash
+       pgai vectorizer worker
+       ```
+
+    You can also use the `-d` or `--db-url` arguments to set a Postgres connection string:
+    ```bash
+    vectorizer -d "postgres://user:password@host:port/dbname"
+    ```
+
+## Run vectorizers with vectorizer worker
+
+By default, when you run a vectorizer worker, it loops over the vectorizers defined in 
+your database and processes each vectorizer in turn. Five minutes after completing each 
+vectorizer run, the vectorizer worker loops over the vectorizers again. 
+For a [local installation](#install-vectorizer-worker-in-your-local-environment-), you use the 
+`-i` / `--vectorizer-id` command line argument to manage which vectorizers that are run by that
+worker instance. For `docker compose` you add arguments using either the `command` or `environment`
+flags in `docker-compose.yml`. 
+
+A vectorizer worker can:
+
+- Run a single vectorizer:
+
+  To run the vectorizer with id 42:
+  - local: `pgai vectorizer worker -i 42`
+  - Docker: `command: ["-i", "42"]`
```

Comment:
This section for `Docker` it's actually docker compose.

I'd change that to docker-compose. 

Docker could be something like `docker run timescale/pgai-vectorizer-worker:{tag version} -i 42`

Reply:
Added the docker run commands.